### PR TITLE
Updated README.md

### LINKED ISSUE #148: Exclude other compiled/generated renpy files from the workspace folder
### Discussed in https://github.com/LuqueDaniel/vscode-language-renpy/discussions/147

There are some more files that should be hidden. Maybe the setting should be changed to 'excludeCompiledRenpyFilesFromWorkspace'.

As an example, along with `.rpyc` there is  `.rpa` and `.rpymc`.

We then might also want to exclude the `cache` folder

### PR DESCRIPTION
**Pull request by KiloOscarSix**
closes LuqueDaniel#145
Made note that the latest version of renpy includes VSCode

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,10 +62,14 @@ Feel free to [contribute](https://github.com/LuqueDaniel/vscode-language-renpy/b
...
 
-## Integrate VSCode as code editor in Ren'Py
+## Integrate VSCode as code editor in Ren'Py (Ren'Py 7.4 and older)
+
+Note: In Ren'py 7.5/8.0 VSCode is already included by default in the list of text editors
 
 You can copy the [`Visual Studio Code.edit.py`](https://raw.githubusercontent.com/LuqueDaniel/vscode-language-renpy/master/Visual%20Studio%20Code.edit.py) file that is available at the root of this repository to the root of your project directory. This will add VSCode (or VSCodium) to the list of text editors in Ren'Py.
 
+
+
```
Comment (LuqueDaniel) [inline README.md:72]: Remove extra blank line.

**Review by LuqueDaniel — CHANGES_REQUESTED** [review]
Thanks for your contribution.

File: README.md (hunk 1 above)
Comment (KiloOscarSix) [reply README.md:72]: Removed 👍

**Review by LuqueDaniel — APPROVED** [review]
LGTM! 🎉 

Thanks for your contribution.

**Pull request merged**